bridge/qb/client/functions.lua

```diff
@@ -86,56 +121,97 @@ end
 
 -- Getters
 
----@deprecated use GetVehicles from imports/utils.lua
-functions.GetVehicles = GetVehicles
+---@deprecated use the GetGamePool('CVehicle') native directly
+functions.GetVehicles = function()
+    GetGamePool('CVehicle')
+end
 
----@deprecated use GetObjects from imports/utils.lua
-functions.GetObjects = GetObjects
+---@deprecated use the GetGamePool('CObject') native directly
+functions.GetObjects = function()
+    GetGamePool('CObject')
+end
 
----@deprecated use GetPlayersInScope from imports/utils.lua
+---@deprecated use the GetPlayersInScope native directly
 functions.GetPlayers = GetPlayersInScope
 
----@deprecated use GetPeds from imports/utils.lua
-functions.GetPeds = GetPeds
+---@deprecated use the GetGamePool('CPed') native directly
+functions.GetPeds = function()
+    GetGamePool('CPed')
+end
 
----@deprecated use GetClosestPed from imports/utils.lua
+---@deprecated use lib.getClosestPed from ox_lib
 ---Use GetClosestPlayer if wanting to ignore non-player peds
-functions.GetClosestPed = GetClosestPed
+functions.GetClosestPed = lib.getClosestPed
```

Comment:
Ditto, also lib returns different types of results than the original [`QBCore.Functions.GetClosestPed`](https://github.com/qbcore-framework/qb-core/blob/cdd1502f0157fe2c9e1d97ef1b6e79eceb9e6c9f/client/functions.lua#L187) function.

Reply:
Everything else is pretty much done except for this function. Trying to Frankenstein this together is kinda confusing so help on this would be appreciated. Take another pass through as well and make sure I did everything right

Reply:
The one from utils correctly converts it